README.md

```diff
@@ -13,16 +13,29 @@ This repo contains CSI driver for Gluster. The Container Storage Interface (CSI)
 This repository consists of Dockerfile for GlusterFS CSI dirver to build on  CentOS
 distribution. Once you clone the repository, to build the image, run the following command:
 
+1) Get inside the repository directory
+
+```
+[root@localhost]#cd gluster-csi-driver
+```
+2) Compile and create a binary
+
+```
+[root@localhost]#make
 ```
-#cd gluster-csi-driver
-#make
-#docker build -t glusterfs-csi-driver -f pkg/glusterfs/Dockerfile .
+
+3) Build a docker image based on the binary compiled above
+
+```
+[root@localhost]#docker build -t glusterfs-csi-driver -f pkg/glusterfs/Dockerfile .
```

Comment:
Can we move the Dockerfile to the root of the project? That will enable this to be built with buildah as well. Since we're using buildah for GD2, I'd rather use one tool to build across the board.